cpp/benchmarks/filter/minmax_filter.cpp

```diff
@@ -59,7 +59,7 @@ struct benchmark_data<T> {
 
 enum class engine_type : uint8_t { AST = 0, JIT = 1 };
 
-engine_type engine_from_string(std::string_view str)
+static engine_type engine_from_string(std::string_view str)
```

Comment:
Typically we don't use `static` keyword for private function but anonymous namespace.